.github/workflows/e2e-tests-ondemand.yaml

```
@@ -77,7 +77,7 @@ jobs:
           tenant2-apitoken: ${{ secrets.TENANT2_APITOKEN }}
           tenant2-dataingesttoken: ${{ secrets.TENANT2_DATAINGESTTOKEN }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
-      - name: Cluster created successfully
+      - name: Cluster creation status
```

Maybe this would be even more descriptive?
```suggestion
      - name: Determine cluster status
```

---

👍